dashboard/app/helpers/levels_helper.rb

```diff
@@ -50,14 +50,14 @@ def readonly_view_options
     view_options callouts: []
   end
 
-  def set_channel
+  def set_channel(level)
```

Comment:
Basically to fail fast. I don't want it to silently use the wrong level if I missed a set_channel call.